lib/WP_Auth0_Api_Client.php

```diff
@@ -240,29 +240,25 @@ public static function create_client( $domain, $app_token, $name ) {
 		$headers['Authorization'] = "Bearer $app_token";
 		$headers['content-type'] = "application/json";
 
-		$logout_url = home_url();
-
 		$response = wp_remote_post( $endpoint  , array(
 				'method' => 'POST',
 				'headers' => $headers,
 				'body' => json_encode( array(
 						'name' => $name,
 						'callbacks' => array(
-							home_url( '/index.php?auth0=1' ),
-							home_url( '/wp-login.php' )
+							site_url( 'index.php?auth0=1' ),
```

Comment:
So the site_url always includes the trailing slash?

Reply:
@cocojoe - It will correct either way:

```
# site_url( 'index.php' ):
http://localhost/index.php

# site_url( '/index.php' ):
http://localhost/index.php

# home_url( '/index.php' ):
http://localhost/index.php

# home_url( 'index.php' ):
http://localhost/index.php
```